db/schemas/eligibility.py

```diff
@@ -0,0 +1,20 @@
+from db.models import Eligibility
+from db.models.eligibility.eligibility_trail import EligibilityUpdate
+from marshmallow import post_dump
+from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
+
+
+class EligibilitySchema(SQLAlchemyAutoSchema):
+    class Meta:
+        model = Eligibility
+        exclude = ["key"]
+
+    @post_dump
+    def handle_nones(self, data, **kwargs):
+        if data["date_submitted"] is None:
+            data["date_submitted"] = "null"
```

Comment:
A string of null, is this correct? Why a string exactly? 🤔

Reply:
I’d think an actual None would be better than a string of “null” - but not sure if this is used anywhere dependent